src/ServerlessAutoSwagger.ts

```diff
@@ -38,7 +38,7 @@ export default class ServerlessAutoSwagger {
   swagger: Swagger = {
     swagger: '2.0',
     info: {
-      title: '',
+      title: 'Serverless service name',
```

Comment:
I was saying the default title is the service name in serverless, not that we should use `Serverless service name` text as a default.

```suggestion
      title: '',
```